interview-150/80.ts

```diff
@@ -0,0 +1,19 @@
+function removeDuplicatesII(nums: number[]): number {
+    let numOcc = 1
+
+    for (let i = 0; i < nums.length; i++) {
+        let next = i + 1
```

Comment:
Should be a `const` - and might as well just change this to the actual value & not just the index, i.e:
```
const next = nums[i + 1];
```